.cloudbuild/e2e.yaml

```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://json.schemastore.org/cloudbuild.json
 substitutions:
-  _NODE10: node:10.24.1-slim
+  _NODE: node:18-slim
```

Comment:
How come we use Alpine in one context and Debian here?

Reply:
Ah, because I only tested one! We need `ps` available which only the alpine flavour comes with.

Reply:
! You tell the truth.

Reply:
Changed in dc2d0a17e5b32dcfe0f934f088f78741fab35bd0.